chrononaut/__init__.py

```diff
@@ -104,4 +109,6 @@ def create_session(self, options):
     "extra_change_info",
     "rationale",
     "ChrononautException",
+    "MigrateFromHistoryTableOp",
```

Comment:
Nit: I wouldn't include these in the global exports.

Reply:
Without exporting this migration globally, we don't get the automatic alembic support and risk dropping history tables without migrating them first

Reply:
Ah, gross. Well we can do it for now then. Maybe we deprecate/drop it in the next version.